### ISSUE TITLE
Wallet request

### ISSUE THREAD

**Issue by heatedlime** [helium/wallet-app#582]
it’s nice that I can click “max” and send all, but is it possible to also add a “half” button? I pay all my hosts 50/50 split, so it would be super convenient to click one button to send half instead of doing the math and typing it out. 

![IMG_7509](https://github.com/helium/wallet-app/assets/104723888/75577350-ce2d-4433-a1a3-645bfc8a9548)

**Comment by Pawar7349** [comment]
Hi @heatedlime 
I'm interested in working on issue #582 about adding a "half" button next to the "max" button. Is this issue still open? If yes, could you let me know if there are any specific guidelines I should follow?

**Comment by heatedlime** [comment]
> Hi @heatedlime I'm interested in working on issue #582 about adding a "half" button next to the "max" button. Is this issue still open? If yes, could you let me know if there are any specific guidelines I should follow?

Yes it’s still open. This is a feature request that I would like to see so that someone can click a “half” button and send half of whatever tokens they are sending.
